### PR TITLE
Move remaining docs to website

### PR DESCRIPTION
**Pull request by mnonnenmacher**
See the commit messages for details.
Please note that the docs need to be updated which will happen in a separate PR.

### CODE REVIEW

File: docs/architecture/different_tool_versions.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+So, the configuration mechanism needs to be quite flexible as well.
+
+When triggering an ORT run, users must be able to specify information about version constraints in a generic way.
+With labels, ORT (and ORT Server) already supports a mechanism to provide arbitrary data that is available during the whole run.
+So it would be possible to define a specific prefix, e.g., _transport:_, for labels that are intended to be evaluated by transport implementations.
+
+When invoking a worker, the Orchestrator could extract all labels starting with this prefix and add them to the message that triggers the worker.
+For this, the `Message` class would have to be extended.
+
+NOTE:
```
Comment (sschuberth) [inline docs/architecture/different_tool_versions.md]: Should this use `:::note` syntax, while at it?

File: docs/architecture/different_tool_versions.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+    orchestrator ->> transport: Add transport property<br/>kubernetes:imageTagSuffix=java17-latest
+    transport ->> kubernetes: Start job with image<br/>ort-server-analyzer:java17-latest
+```
+
+The flow begins with the user sending a request to trigger a run for a specific repository.
+As described under [Parameters when triggering an ORT run](../../website/docs/guides/parameter-validation-script.md), the API to trigger requests can be customized using a validation script.
+In this example, the script supports a _javaVersion_ parameter in the generic parameters of the Analyzer job configuration.
+The value is the version of the JDK that needs to be installed in the container image running the Analyzer step.
+
+NOTE:
```
Comment (sschuberth) [inline docs/architecture/different_tool_versions.md]: See above.

File: docs/config/config.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
+# Configuration of ORT Server
+
+This document gives an overview over the configuration options supported by ORT Server.
+It also shortly describes the configuration mechanism in place.
+
+NOTE:
```
Comment (sschuberth) [inline docs/config/config.md]: `:::note`?

File: docs/architecture/different_tool_versions.md (hunk 1 above)
Comment (mnonnenmacher) [reply docs/architecture/different_tool_versions.md]: I have removed the "NOTE" altogether, the paragraph does not really stand out to me.

File: docs/architecture/different_tool_versions.md (hunk 2 above)
Comment (mnonnenmacher) [reply docs/architecture/different_tool_versions.md]: I have removed the whole paragraph, it does not match the new diagram.

**Review by sschuberth — APPROVED** [review]

**Pull request merged**